assembly/index.ts

```diff
@@ -1 +1,3 @@
-// TODO: Set up globals, like Buffer
+/// <reference path="../node_modules/assemblyscript/std/types/assembly/index.d.ts" />
```

Comment:
Wondering if it would be better when the `assembly/` directory had a tsconfig.json inheriting the assembly.json from the compiler, as that's the intended mechanism.

Reply:
Ah. That was leftover from the initial commit I made.

Reply:
Just removed it.